packages/core/src/components/forms/_label.scss

```diff
@@ -8,19 +8,41 @@
 /*
 Labels
 
-Add labels to inputs to enhance the usability of your forms.
+Labels enhance the usability of your forms.
 
-Putting the `<input>` element _inside_ a `<label>` element also increases the area where the user
-can click to activate the control. Notice how in the example below, clicking a `<label>` below
-focuses its `<input>`.
+<div class="pt-callout pt-intent-success pt-icon-comparison">
+  <h5>Simple labels vs. form groups</h5>
+  <p>Blueprint provides two ways of connecting label text to control fields, depending on the complexity of the control.</p>
+  <p>Simple labels are a basic way to connect a label with a single control.</p>
+  <p>Form groups require more markup to maintain consistent visuals across more complex controls.</p>
+</div>
+
+Weight: 1
+
+Styleguide components.forms.labels
+*/
+
+/*
+Simple labels
+
+Simple labels are useful for basic forms eg. for a single `<input>`. This structure is not sufficient for some complex form fields such as
+[numeric inputs](#components.forms.numeric-input) or [control groups](#components.forms.control-group). These require more complicated
+[form groups](#components.forms.labels.form-groups).
```

Comment:
i don't think these last two sentences are helpful here cuz they're repeated elsewhere in the docs. please remove so it's just about simple labels.